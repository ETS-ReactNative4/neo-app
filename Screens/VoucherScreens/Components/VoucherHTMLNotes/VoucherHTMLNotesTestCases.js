import React from "react";
import VoucherHTMLNotes from "./VoucherHTMLNotes";

const htmlData = `<div>
  <ul>
    <li>Upon arrival, phone 0800 935050 and let us know that you are ready to be picked up (Airport free phones are located near the baggage collection area)</li>
    <li>Upon arrival, phone 0800 935050 and let us know that you are ready to be picked up (Airport free phones are located near the baggage collection area)</li>
    <li>Upon arrival, phone 0800 935050 and let us know that you are ready to be picked up (Airport free phones are located near the baggage collection area)</li>
    <li>Upon arrival, phone 0800 935050 and let us know that you are ready to be picked up (Airport free phones are located near the baggage collection area)</li>
    <li>Upon arrival, phone 0800 935050 and let us know that you are ready to be picked up (Airport free phones are located near the baggage collection area)</li>
  </ul>
</div>`;

const VoucherHTMLNotesTestCases = [
  {
    title: "Voucher HTML notes data",
    Component: <VoucherHTMLNotes data={htmlData} />
  }
];

export default VoucherHTMLNotesTestCases;
