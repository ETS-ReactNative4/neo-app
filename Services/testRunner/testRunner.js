import renderer from "react-test-renderer";

const testRunner = testCase => {
  test(testCase.title, () => {
    const tree = renderer.create(testCase.Component).toJSON();
    expect(tree).toMatchSnapshot();
  });
};

export default testRunner;
