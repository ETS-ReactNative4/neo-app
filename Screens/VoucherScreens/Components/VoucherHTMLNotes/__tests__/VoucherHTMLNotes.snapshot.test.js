import VoucherHTMLNotesTestCases from "../VoucherHTMLNotesTestCases";
import testRunner from "../../../../../Services/testRunner/testRunner";

VoucherHTMLNotesTestCases.map(testRunner);
