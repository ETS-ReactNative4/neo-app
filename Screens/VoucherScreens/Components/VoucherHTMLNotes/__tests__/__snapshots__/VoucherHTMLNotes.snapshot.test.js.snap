// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Voucher HTML notes data 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "silver",
        "flex": 1,
        "marginTop": 48,
      },
      Object {},
    ]
  }
>
  <View>
    <Text
      onLongPress={null}
      onPress={null}
    >
      <Text
        style={
          Array [
            null,
            Object {},
          ]
        }
      >
          
      </Text>
      <View
        style={
          Object {
            "alignItems": "flex-start",
            "flexDirection": "row",
            "justifyContent": "flex-start",
          }
        }
      >
        <View
          style={
            Object {
              "marginTop": 4,
            }
          }
        >
          <Text
            allowFontScaling={false}
            style={
              Array [
                Object {
                  "color": "rgba(255, 201, 51, 1)",
                  "fontSize": 15,
                },
                undefined,
                Object {
                  "fontFamily": "voyager",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
                Object {},
              ]
            }
          >
            î¥ž
          </Text>
        </View>
        <View>
          <Text
            style={
              Object {
                "color": "rgba(44, 47, 66, 1)",
                "fontFamily": "SourceSansPro-Regular",
                "fontSize": 14,
                "lineHeight": 22,
                "marginLeft": 4,
              }
            }
          >
            Some Text Goes here
          </Text>
        </View>
      </View>
    </Text>
  </View>
</View>
`;
